scripting/get5.sp

```diff
@@ -522,20 +522,24 @@ public Action Timer_InfoMessages(Handle timer) {
   return Plugin_Continue;
 }
 
+public void AdminClientKickCheck(int client) {
+  if (IsPlayer(client)) {
+    if (g_GameState == Get5State_None && g_KickClientsWithNoMatchCvar.BoolValue) {
+      if (!g_KickClientImmunity.BoolValue ||
+          !CheckCommandAccess(client, "get5_kickcheck", ADMFLAG_CHANGEMAP)) {
+            KickClient(client, "%t", "NoMatchSetupInfoMessage");
+      }
+    }
+  }
+}
```

Comment:
Can you just move this into `OnClientPostAdminCheck`? The extra function seems unnecessary.